src/Symfony/Bundle/WebServerBundle/WebServer.php

```diff
@@ -145,12 +144,9 @@ public function isRunning($pidFile = null)
      */
     private function createServerProcess(WebServerConfig $config)
     {
-        $finder = new PhpExecutableFinder();
-        if (false === $binary = $finder->find()) {
-            throw new \RuntimeException('Unable to find the PHP binary.');
-        }
+        $executable = $config->getExecutable();
 
-        $process = new Process(array($binary, '-S', $config->getAddress(), $config->getRouter()));
+        $process = new Process(array($executable, '-S', $config->getAddress(), $config->getRouter()));
```

Comment:
This is not as simple as it seems. Consider following executable: `php -dmemory_limit=16M -dzend_extension="C:\Program Files\some_extension.dll"`. If I were to split this with spaces, it would break around the `Program Files` part. And parsing CLI arguments is way over the scope of this PR IMHO. I've googled for some ready-made solutions and everything is in the ballpark of hundred lines of code. 

I tried to repurpose `\Symfony\Component\Console\Input\StringInput` for this, but it feels very hacky and I needed to add new public method to it. 

See 50fda2378283319bfe6e3c6da28bfce39aef18d9

Tested with ` php index.php s:r --docroot web\  --executable "php -dmemory_limit=512M -dinclude_path=\"c:\Program Files\""`